bioconda_utils/pkg_test.py

```diff
@@ -83,7 +83,7 @@ def get_image_name(path):
 def test_package(
     path,
     name_override=None,
-    channels=["conda-forge", "defaults"],
+    channels=("conda-forge", "bioconda", "local", "defaults"),
```

Comment:
The order `"bioconda", "local"` is what we currently have (since `bioconda` gets prepended to everything). Granted, that order is debatable. What we *actually* would want is to have `bioconda` and `local` at the same priority. `conda` currently does not provide such functionality. If we wanted to have that, we would either have to
1. put such functionality into `conda`, or
2. spawn a local HTTP server that transparently merges `bioconda` and `local`.

The second option might be easier since for the first option one would have to think about and coordinate the user interface changes with the Conda devs.

Reply:
I would think it should be local first, shouldn't it? At least before bioconda. Because those are the packages that where created during that run, and we surely want to prefer those over old stuff in bioconda.

Reply:
Merging `bioconda` and `local` would replicate what you get after the packages have been uploaded, so I think this would be the "ideal" solution, maybe.
`(bioconda, local)` is just the status quo, but I completely agree that `(local, bioconda)` is the better choice.